exp-player/addon/components/exp-lookit-story-page/component.js

```diff
@@ -336,6 +347,16 @@ export default ExpFrameBaseUnsafeComponent.extend(FullScreen, VideoRecord,  {
         }
     },
 
+    parentTextStyle: Ember.computed('parentTextBlock', function() {
+        var parentTextBlock = this.get('parentTextBlock') || {};
+        var css = parentTextBlock.css || {};
+        var cssString = '';
+        $.each(css, function(propName, value) {
+            cssString += new Ember.Handlebars.SafeString(`${propName}:  ${value};`);
```

Comment:
Whoops, I was taken in by the name. jquery css would work fine! Would it be okay to do something like `$('#parenttext').css(propName, value);` in place of building the `cssString`?